WooCommerce/Classes/System/SessionManager.swift

```diff
@@ -95,6 +95,11 @@ final class SessionManager: SessionManagerProtocol {
         }
     }
 
+    /// Unique WooCommerce Store UUID.
+    /// Do not conduse with `defaultStoreID` which is in fact the WPCom `siteID`.
```

Comment:
super nit: how about also mentioning the purpose of this store ID (for merging store data from different platforms)?
```suggestion
    /// Do not confuse with `defaultStoreID` which is in fact the WPCom `siteID`.
```